pylint_django/transforms/foreignkey.py

```diff
@@ -41,6 +41,18 @@ def _get_model_class_defs_from_module(module, model_name, module_name):
     return class_defs
 
 
+def _module_name_from_django_model_resolution(model_name, module_name):
+    import django  # pylint: disable=import-outside-toplevel
+    django.setup()
+    from django.apps import apps  # pylint: disable=import-outside-toplevel
```

Comment:
same for this one, top-level ?

Reply:
Oh good catch. This was left over from before when I was checking to see if Django was installed. Will fix.